app/views/laboratory_samples/_filters.haml

```diff
@@ -18,4 +18,3 @@
             = cdx_select name: :sample_type, value: params["sample_type"], searchable: false do |select|
               - select.item "", "Show all"
               - select.items LaboratorySample.sample_types
-
```

Comment:
Let's pay attention to this whitespace changes. If the change isn't meaningful, let's not add them to the commit.